client/whatami/whatami

```diff
@@ -219,6 +222,11 @@ get_linux_type()
                         #distro=${distro_brand}_${distro_version}
                         distro=${distro_brand}
 
+                elif [ -f /etc/redhat-release ]; then
```

Comment:
I want to add a further check to this statement to make sure it contains the correct string.

Something like:
```bash
elif [ -f /etc/redhat-release -a -n "`egrep 'Red Hat Enterprise Linux ([a-zA-Z]+) release [0-9]*' /etc/redhat-release`" ]; then
```

I'll make the update shortly.